sylviawanjiku/StoreManager
# #161164993 View an individual product minimum inventory quantity allowed at any time.

#### What does this PR do?
Allows the user to view an individual product minimum inventory quantity allowed at any time
#### Description of Task to be completed?
A user can view an individual product minimum inventory quantity allowed from the products list and in case the stock available for the product is less than the minimum stock allowed the user can note easily from the change in the color of the text for that product 
#### How should this be manually tested?
A user can reduce the stock available to less than the minimum stock allowed and check to see if the text color changes
#### Any background context you want to provide?
As long as the stock is equal or less than the minimum quantity it should automatically change the color of the text
#### What are the relevant pivotal tracker stories?
#161164993
https://www.pivotaltracker.com/story/show/161164993
#### Screenshots (if appropriate)
![screenshot_2018-10-12 store manager products 1](https://user-images.githubusercontent.com/28457081/46846174-5317ac80-cde7-11e8-97aa-30e07d12f538.png)

#### Questions:
